assets/components/ovn/configmap.yaml

```diff
@@ -29,6 +29,7 @@ data:
     [gateway]
     mode=local
     nodeport=true
+    single-node=true
```

Comment:
Is this asset managed by the rebase script?

Reply:
Not yet.
The ovnk assets commands are currently commented out in the rebase script. 
Adding microshift ovnk manifests to openshift repo doesn't add any extra value for us now since there is no auto-generation of manifests nor CI tests in openshift repo.